diff --git a/lua/core/plugins.lua b/lua/core/plugins.lua
index c8b2c68..62c8837 100644
--- a/lua/core/plugins.lua
+++ b/lua/core/plugins.lua
@@ -30,14 +30,6 @@ local astro_plugins = {
   -- Neovim UI Enhancer
   ["MunifTanjim/nui.nvim"] = { module = "nui" },
 
-  -- Cursorhold fix
-  ["antoinemadec/FixCursorHold.nvim"] = {
-    event = { "BufRead", "BufNewFile" },
-    config = function()
-      vim.g.cursorhold_updatetime = 100
-    end,
-  },
-
   -- Smarter Splits
   ["mrjones2014/smart-splits.nvim"] = {
     module = "smart-splits",
@@ -129,7 +121,6 @@ local astro_plugins = {
 
   -- Completion engine
   ["hrsh7th/nvim-cmp"] = {
-    event = "InsertEnter",
     config = function()
       require "configs.cmp"
     end,
